app/views/admin/_revisions.html.erb

```diff
@@ -16,7 +16,9 @@
         <%= revision.body.truncate(40) %>
       </td>
       <td style="width:200px;">
+      <% unless revision.author.nil? %>
         <a href="/profile/<%= revision.author.name %>"><%= revision.author.name %></a> 
```

Comment:
Here, I propose:
```suggestion
        <a href="/profile/<%= revision.author&.name %>"><%= revision.author&.name %></a> 
```
instead of using the `unless` -- because then it'll be simpler than the conditional logic, and will fail gracefully! What do you think for this and other instances here?

Reply:
Sure, It will work here.  So, should I change in all the places of /spam and /spam2 where conditional logic is used??

Reply:
@jywarren I have made changes as suggested. Please review!
Thanks!!